data_scripts/createElasticIndex.js

```diff
@@ -2,7 +2,7 @@ var elasticsearch = require('elasticsearch')
 var async = require('async')
 
 var CLIENT = new elasticsearch.Client({
-    host: 'localhost:9200',
```

Comment:
this *might break current GN

Reply:
Should I change it back? Without this change docker won't work

Reply:
I will test this locally and get back to you